src/core/mocks/domains/language.handlers.ts

```diff
@@ -22,6 +22,11 @@ export const handlers = [
 		return res(ctx.status(200), ctx.json(response));
 	}),
 
+	rest.get(umbracoPath('/languages'), (req, res, ctx) => {
```

Comment:
This endpoint doesn't seem to exist on the language controller. What is it trying to do?

Reply:
I have updated the code to use the correct culture resource. I will get this PR merged.